chart/jenkins-operator/values.yaml

```diff
@@ -184,6 +184,16 @@ jenkins:
     # secretData creates new secret if secretRefName is empty and fills with data provided in secretData
     secretData: {}
 
+  services:
+    http:
+      port: 8080
```

Comment:
https://jenkinsci.github.io/kubernetes-operator/docs/getting-started/latest/schema/#github.com/jenkinsci/kubernetes-operator/pkg/apis/jenkins/v1alpha2.Service there are much more field to map